embc-app/ClientApp/src/app/self-registration/self-registration-one/self-registration-one.component.ts

```diff
@@ -307,6 +316,8 @@ export class SelfRegistrationOneComponent implements OnInit, OnDestroy {
             province: primaryResidence.province,
             country: primaryResidence.country,
           },
+          // if we got a primary address and no mailing address, it means they are the same.
+          mailingAddressSameAsPrimary: mailingAddress == null,
```

Comment:
Don't need to fix this but I would prefer to see
`mailingAddressSameAsPrimary: (mailingAddress == null),`
to clearly show that the property value is a calculation.

Reply:
Good catch, done!